javascript-typing-tutor/main.js

```diff
@@ -0,0 +1,29 @@
+
+var span = document.querySelectorAll('span');
+document.addEventListener('keydown', checkLetter);
+
+function checkLetter(event) {
+
+  for (var i = 0; i < span.length; i++) {
+    if (span[i].className === 'next') {
+      var currentLetter = span[i].innerText;
+      if (event.key !== currentLetter) {
+        span[i].className = 'incorrect';
+      } else {
+        span[i].className = 'correct';
+        span[i + 1].className = 'next';
+      }
+      return;
+    }
+    if (span[i].className === 'incorrect') {
+      currentLetter = span[i].innerText;
+      if (event.key !== currentLetter) {
+        span[i].className = 'incorrect';
+      } else {
+        span[i].className = 'correct';
+        span[i + 1].className = 'next';
+      }
+      return;
+    }
```

Comment:
The work for both of these conditionals seems to be the same, you can likely use on conditional with an `||` operator in the condition.